portfolio/src/main/webapp/script.js

```diff
@@ -32,6 +32,12 @@ async function fetchServerMessage() {
     const textResponse = await serverResponse.text();
 
     const msgContainer = document.getElementById('msg-container');
-    msgContainer.innerText = textResponse;
+    const commentArrString = textResponse.substring(textResponse.indexOf('[')+1,textResponse.indexOf(']'));
```

Comment:
The response is a JSON object, which is technically a string but can be parsed to make it into a concrete type. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse